plugins/juju/README.md

```diff
@@ -38,6 +38,7 @@ Naming convention:
 
 | Alias    | Command                                                                               | Description                                                       |
 |----------|---------------------------------------------------------------------------------------|-------------------------------------------------------------------|
+| `jctl`   | `juju controllers`                                                                    | List all controllers                                              |
```

Comment:
This alias is not actually added in the code.

Reply:
Added